cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/Cargo.toml

```diff
@@ -37,3 +41,17 @@ cumulus-pallet-dmp-queue = { path = "../../../../../../pallets/dmp-queue", defau
 bridge-hub-rococo-runtime = { path = "../../../../../../parachains/runtimes/bridge-hubs/bridge-hub-rococo", default-features = false }
 emulated-integration-tests-common = { path = "../../../common", default-features = false }
 rococo-westend-system-emulated-network = { path = "../../../networks/rococo-westend-system" }
+bridge-hub-rococo-emulated-chain = { path = "../../../chains/parachains/bridges/bridge-hub-rococo" }
+asset-hub-rococo-emulated-chain = { path = "../../../chains/parachains/assets/asset-hub-rococo" }
```

Comment:
```suggestion
```
@claravanstaden  unnecessary  import, you can just click "apply this suggestion" (I tried locally and cannot push to your fork)